tendrl/ceph_integration/manager/crud.py

```diff
@@ -119,3 +123,14 @@ def delete(self, object_type, object_id):
             )
         else:
             raise NotImplementedError(object_type)
+
+    def sync_request_status(self, request):
+        count = 0
+        while count < 30:
```

Comment:
@r0h4n request state is changed by next heartbeat time out and that takes few mins. I feel keeping 5 mins is safe and with smaller value we always see pool creation success in backend but request timed out